src/passkit.cs

```diff
@@ -1171,27 +1213,29 @@ interface PKLabeledValue
 
 	[Mac (11,0)]
 	[Watch (4,3), iOS (11,3)]
-	[BaseType (typeof (NSObject))]
+	[BaseType (typeof (PKStoredValuePassProperties))]
 	[DisableDefaultCtor]
 	interface PKTransitPassProperties {
 
-		[Static]
-		[Export ("passPropertiesForPass:")]
-		[return: NullAllowed]
-		PKTransitPassProperties GetPassProperties (PKPass pass);
-
+		[Deprecated (PlatformName.iOS, 15, 0)]
+		[Deprecated (PlatformName.WatchOS, 8, 0)]
+		[Deprecated (PlatformName.MacOSX, 12, 0)]
 		[Export ("transitBalance", ArgumentSemantic.Copy)]
 		NSDecimalNumber TransitBalance { get; }
 
+		[Deprecated (PlatformName.iOS, 15, 0)]
+		[Deprecated (PlatformName.WatchOS, 8, 0)]
+		[Deprecated (PlatformName.MacOSX, 12, 0)]
 		[Export ("transitBalanceCurrencyCode")]
 		string TransitBalanceCurrencyCode { get; }
 
 		[Export ("inStation")]
 		bool InStation { [Bind ("isInStation")] get; }
 
-		[Deprecated (PlatformName.iOS, 14,5, message: "Use 'Blocked' instead.")]
-		[Deprecated (PlatformName.MacCatalyst, 14,5, message: "Use 'Blocked' instead.")]
-		[Deprecated (PlatformName.WatchOS, 7,4, message: "Use 'Blocked' instead.")]
+		[Deprecated (PlatformName.iOS, 15, 0, message: "Use 'Blocked' instead.")]
+		[Deprecated (PlatformName.MacCatalyst, 15, 0, message: "Use 'Blocked' instead.")]
+		[Deprecated (PlatformName.WatchOS, 8, 0, message: "Use 'Blocked' instead.")]
+		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use 'Blocked' instead.")]
```

Comment:
`Blocked` availability was earlier (see below) so the deprecated should not change - except that macOS should have one for 11,3

Reply:
ah, looks like I got confused with a [different blocked](https://developer.apple.com/documentation/passkit/pkstoredvaluepassproperties/3747127-blocked?changes=_1&language=objc) - thanks for catching!